modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/ImageMosaicFormat.java

```diff
@@ -227,6 +228,15 @@
                     new ExcessGranulePolicy[] {ExcessGranulePolicy.NONE, ExcessGranulePolicy.ROI},
                     ExcessGranulePolicy.NONE);
 
+    /** mask (as a polygon in native coordinates) to be applied to the produced mosaic */
```

Comment:
At the moment, it is not supported for Heterogeneous CRS mosaic. There is a Warning log here:
https://github.com/geotools/geotools/pull/1899/commits/5aad04ccc379f9a89e57779630ad2070deb71de2#diff-008e96ae839fc2ef4b56fcfebd744f05R373

and this will be documented.